integration-tests/physics/test_terminator_toy.py

```diff
@@ -0,0 +1,112 @@
+"""
+This tests the terminator toy physics scheme that models the interaction
+of two chemical species through coupled ODEs.
+"""
+
+from gusto import *
+from firedrake import IcosahedralSphereMesh, Constant, cos, \
+    sin, SpatialCoordinate, Function, max_value, as_vector, \
+    errornorm, norm
+import numpy as np
+
+
+def run_terminator_toy(dirname):
+
+    # ------------------------------------------------------------------------ #
+    # Set up model objects
+    # ------------------------------------------------------------------------ #
+
+    # A much larger timestep than in proper simulations, as this
+    # tests moving towards a steady state with no flow.
+    dt = 100000.
+
+    # Make the mesh and domain
+    R = 6371220.
+    mesh = IcosahedralSphereMesh(radius=R,
+                                 refinement_level=3, degree=2)

```

Comment:
As this is just a test and the spatial discretisation is not important, could we use a coarser grid? e.g. refinement level 2?

Reply:
That's a good point. Using a refinement level of 2 still successfully tests the ODEs moving the system towards the steady state, although I will need to halve the timestep size and slacken the tolerance on the test (error less than 0.4) to adjust.